configure

```diff
@@ -353,6 +353,27 @@ EOF
   cc_statement HAVE_STAT_TV_NSEC2 \
     'struct stat st; st.st_mtimespec.tv_nsec = 1; return 0;' \
     '#include <sys/stat.h>'
+
+  if test "$FLAG_without_readline" = 1; then
```

Comment:
OK just reminding myself how this works ... right now `_build/detected-config.sh` has HAVE_READLINE=1, and is used by `build/ovm-compile.sh`

Eventually I think we get rid of that ...